e2e/node/e2e.test.ts

```diff
@@ -366,19 +365,28 @@ const serversUnderTest: AuthDetails[] = [
 // eslint-disable-next-line jest/no-disabled-tests
 describe.each(serversUnderTest)(
   "Authenticated end-to-end tests against Pod [%s] and OIDC Issuer [%s]:",
-  (rootContainer, oidcIssuer, clientId, clientSecret) => {
+  (rawRootContainer, rawOidcIssuer, clientId, clientSecret) => {
+    if (!rawRootContainer || !rawOidcIssuer) {
+      console.log("Skipping test due to missing env variables");
+      return;
+    }
```

Comment:
We really need to address the test config, see: https://github.com/inrupt/solid-client-js/pull/1369#issuecomment-983607583

Right now we've three different sets of env vars for the tests; It'd be far better to have just one set.